runit-elixir/Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
 # Any changes to these versions should be synchronized in the
 # jenkins-workers cookbook and a new AMI should be built.
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
-    DEBIAN_FRONTEND=noninteractive apt-get -y install esl-erlang=1:23.0.* elixir=1.10.*
+    DEBIAN_FRONTEND=noninteractive apt-get -y install esl-erlang=1:24.0.6-1 elixir=1.12.*
```

Comment:
Is there any particular reason we're specifying the patch version? I think we'd want it to float, for security reasons, right?

Reply:
sure, we can do that